package.json

```diff
@@ -17,8 +17,8 @@
     "hashtrack": "^1.0.2",
     "underscore": "^1.7.0",
     "jquery-form": "~3.50.0",
-    "OverlappingMarkerSpiderfier": "yagoferrer/OverlappingMarkerSpiderfier#747476d7bf",
-    "Flot": "flot/flot#453b017cc5"
+    "marker-spider": "0.1.0",
```

Comment:
Here's the doc on the version string: https://docs.npmjs.com/files/package.json#dependencies

So, it seems that not having a punctuation prefix means 'exact', whereas the other punctuation prefices are less strict, so I'd prefer to keep it as is.